Tests/js/models/assets/ez-contentmodel-tests.js

```diff
@@ -388,6 +417,8 @@ YUI.add('ez-contentmodel-tests', function (Y) {
         setUp: function () {
             var that = this;
 
+            Y.eZ.Version = Y.Base.create('versionModel', Y.eZ.RestModel, [], {});
```

Comment:
I have added `ez-versionmodel` dependancy for `ez-contentmodel` test in `Tests/js/models/ez-contentmodel.html`